src/MCPClient/lib/clientScripts/post_store_aip_hook.py

```diff
@@ -122,7 +143,9 @@ def post_store_hook(sip_uuid):
             elasticSearchFunctions.remove_transfer_files(client, transfer_uuid)
 
     # DSPACE HANDLE TO ARCHIVESSPACE
-    dspace_handle_to_archivesspace(sip_uuid)
+#    dspace_handle_to_archivesspace(sip_uuid)
```

Comment:
Why does it need to be commented out? It looks like it's a no-op when the ArchivesSpaceDigitalObject object doesn't exist. Isn't that the case here?

Reply:
Just taking Justin's advice, since dspace will not be used in Jisc.
If it's just configured out, that's better, especially if we are keeping forks close.